### ISSUE TITLE
[FR] Quickly adjust already added video timestamps via commands.

### ISSUE THREAD

**Issue by XMB-7** [PKM-er/media-extended#224]
On youtube or bilibili, the smallest granularity of a timestamp is 1 second, but for media-extended it is much finer, so we could have something like `[00:03](https://youtu.be/qB4HGMvrhwE#t=3.239136)` instead of `#t=3 or #t=4`, providing a much more precise link to the exact audio/video position we'd intended to reference.

To utilize this feature, commands for quick adjustments of added video timestamps are needed. For example, executing a command would automatically decrease the timestamp in the link where the cursor just locates or hovers by a customizable time interval, such as 0.25s, 0.5s or 1s.

In other words, when continuously executing such a command, it would be like:
`[00:39](https://youtu.be/qB4HGMvrhwE#t=39)` -> `[00:39](https://youtu.be/qB4HGMvrhwE#t=38.75)` -> `[00:39](https://youtu.be/qB4HGMvrhwE#t=38.5)` -> `[00:38](https://youtu.be/qB4HGMvrhwE#t=38.25)` -> `[00:38](https://youtu.be/qB4HGMvrhwE#t=38)`.
